_posts/2021-12-31-a-year-in-review-2021.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: epic
+title: "A Year in Review: 2021"
+date: 2021-12-31
+categories: [team, engineering]
+author: artsy_engineering
+---
+
+Wow - what a year! Here are some notable achievements, failures, and learnings from Artsy Engineering in 2021...
+
+<!-- more -->
+
+## Success!
+
+- Leveraged our investment in React Native to launch our Android App: Props to David, Mounir, Pavlos, Joanna, and the Collector Experience team
+- Moved X% of Force apps to V2 and Y% of Force traffic: Props to Chris, Damon, (...)
+- We killed X repos and retired N services
+- The awards for fastest PR from start date goes to: A! And even better, Ole did his first PR to Artsy 3 months before he joined.
```

Comment:
I'm not a huge fan of glorifying "fastest to contribute," given we emphasize that sprint contribution isn't the goal of on-boarding rotations.

Reply:
Agreed but nice to highlight people who contributed to Artsy before joining!

Reply:
Agree with both points. Seeing folks dive in like this _can_ be encouraging for other devs though, just in the sense that it shows we're receptive to initiative.